<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGBA::VideoView</class>
 <widget class="QWidget" name="QGBA::VideoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>593</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Record Video</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="start">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="stop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="selectFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLineEdit" name="filename">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Presets</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="presetHQ">
          <property name="text">
           <string>High &amp;Quality</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">presets</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="presetYoutube">
          <property name="text">
           <string>&amp;YouTube</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">presets</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="presetWebM">
          <property name="text">
           <string>WebM</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">presets</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="presetMP4">
          <property name="text">
           <string>MP4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="presetLossless">
          <property name="text">
           <string>&amp;Lossless</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">presets</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="preset4K">
          <property name="text">
           <string>4K</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="preset1080">
          <property name="text">
           <string>&amp;1080p</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">resolutions</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="preset720">
          <property name="text">
           <string>&amp;720p</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">resolutions</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="preset480">
          <property name="text">
           <string>&amp;480p</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">resolutions</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="presetNative">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Native</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">resolutions</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="advancedBox" native="true">
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="formatBox">
        <property name="title">
         <string>Format</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QComboBox" name="container">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>MKV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WebM</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AVI</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MP4</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="video">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>H.264</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>H.264 (NVENC)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>HEVC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>HEVC (NVENC)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>VP8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>VP9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FFV1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="audio">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>FLAC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WavPack</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Opus</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vorbis</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MP3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AAC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Uncompressed</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="bitrateBox">
        <property name="title">
         <string> Bitrate (kbps)</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QRadioButton" name="doVbr">
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">ratefactor</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QSpinBox" name="abr">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>16</number>
           </property>
           <property name="maximum">
            <number>320</number>
           </property>
           <property name="value">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="doCrf">
           <property name="text">
            <string notr="true"/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">ratefactor</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>ABR</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>abr</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="crf">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
           <property name="value">
            <number>18</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="vbr">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>VBR</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>vbr</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>CRF</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>crf</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="dimensionsBox">
        <property name="title">
         <string>Dimensions</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string notr="true">:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string notr="true">×</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="height">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>3160</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="width">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>3840</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="hratio">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="wratio">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QCheckBox" name="lockRatio">
           <property name="text">
            <string>Lock aspect ratio</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="showAdvanced">
       <property name="text">
        <string>Show advanced</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filename</tabstop>
  <tabstop>start</tabstop>
  <tabstop>stop</tabstop>
  <tabstop>selectFile</tabstop>
  <tabstop>presetHQ</tabstop>
  <tabstop>presetYoutube</tabstop>
  <tabstop>presetWebM</tabstop>
  <tabstop>presetMP4</tabstop>
  <tabstop>presetLossless</tabstop>
  <tabstop>preset4K</tabstop>
  <tabstop>preset1080</tabstop>
  <tabstop>preset720</tabstop>
  <tabstop>preset480</tabstop>
  <tabstop>presetNative</tabstop>
  <tabstop>showAdvanced</tabstop>
  <tabstop>container</tabstop>
  <tabstop>video</tabstop>
  <tabstop>audio</tabstop>
  <tabstop>doCrf</tabstop>
  <tabstop>doVbr</tabstop>
  <tabstop>crf</tabstop>
  <tabstop>vbr</tabstop>
  <tabstop>abr</tabstop>
  <tabstop>width</tabstop>
  <tabstop>height</tabstop>
  <tabstop>wratio</tabstop>
  <tabstop>hratio</tabstop>
  <tabstop>lockRatio</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>doCrf</sender>
   <signal>toggled(bool)</signal>
   <receiver>crf</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doVbr</sender>
   <signal>toggled(bool)</signal>
   <receiver>vbr</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="presets"/>
  <buttongroup name="resolutions"/>
  <buttongroup name="ratefactor"/>
 </buttongroups>
</ui>
